README.md

```diff
@@ -1,6 +1,197 @@
 # Portals
-Portals is a very early proposal for enabling seamless navigations between sites or pages. In particular, this proposal enables a page to show another page as an inset and perform a seamless transition between an inset state and a navigated state.
 
-See the **[explainer](https://github.com/WICG/portals/blob/master/explainer.md)**.
+Portals are a proposed new HTML element which enables seamless navigations between pages. In particular, the `<portal>` element enables a page to show another page as an inset, and then "activate" it to perform a seamless transition to a new state, where the formerly-inset page becomes the top-level document.
```

Comment:
> becomes the top-level document.

Does this language work if the portal is activating within an iframe?

Reply:
Portals cannot load inside an iframe.

Reply:
Ohh I didn't realise that. Hah.